Modules/HypnokinkModule/Attacks/HK_ReciteMantra.gd

```diff
@@ -0,0 +1,50 @@
+extends Attack
+
+func _init():
+	id = "ReciteMantra"
+	category = Category.Special
+	aiCategory = AICategory.Unspecified
+	
+func getVisibleName(_context = {}):
+	return "Recite Mantra"
+	
+func getVisibleDesc(_context = {}):
+	return "Recite a short mantra to regain stamina up to half your hypnosis level."
```

Comment:
A bit confusing. Maybe it'd be better if it was
"Recite a short mantra to regain X-X stamina, scales with your hypnosis skill."
Yeah, you will have to calculate how much (best if you make a function out of it or something)

Reply:
Added actual numbers, clarified text a bit.